applications/rag/README.md

```diff
@@ -182,7 +185,7 @@ This step generates the vector embeddings for your input dataset. Currently, the
 2. From [Google Cloud Platform IAP](https://pantheon.corp.google.com/security/iap), check if the allowlisted user has role `IAP-secured Web App User`. This role is necessary to access the application through IAP.
```

Comment:
Explain who the allowlisted user is and how to check the allowlisted user has the IAP-secured Web App User role. IAM lingo is difficult (even for me) to understand at times.